dp/cloud/python/magma/test_runner/pytest.ini

```diff
@@ -15,3 +15,5 @@ markers =
     local: integration tests without orc8r
 log_cli = True
 log_level = DEBUG
+log_format = %(asctime)s %(levelname)s %(message)s
```

Comment:
do we need both this and k8s timestamps?

Reply:
I reverted this change

Reply:
hmm, I'm actually wondering it would better if we had only that change instead of kubeclt one, since it would only in one place (so it would work for both orc8r tests and without orc8r test) and be consistent with the other components which use their own timestamps instead of relying on k8s